ebos/eclwriter.hh

```diff
@@ -476,7 +480,7 @@ private:
 
     Opm::data::Solution computeTrans_(const std::unordered_map<int,int>& cartesianToActive) const
     {
-        const auto& cartMapper = simulator_.vanguard().cartesianIndexMapper();
+        const auto& cartMapper = simulator_.vanguard().equilCartesianIndexMapper();
```

Comment:
@totto82: I think this is a potential bug, but I'm not 100% sure.

Reply:
Yes that seems like a bug. The cartesian indices are used to check whether all cells between vertical neighbors are inactive (if not it would be an NNC).

Reply:
```
        const auto& cartMapper = simulator_.vanguard().cartesianIndexMapper();
        const auto& cartDims = cartMapper.cartesianDimensions();
        const int globalSize = cartDims[0]*cartDims[1]*cartDims[2];
```
It is only used to get the globalSize. That is the same for both IndexMappers I think.